src/Components/Patient/DailyRoundListDetails.tsx

```diff
@@ -11,23 +11,29 @@ const Loading = lazy(() => import("../Common/Loading"));
 const symptomChoices = [...SYMPTOM_CHOICES];
 const currentHealthChoices = [...CURRENT_HEALTH_CHANGE];
 
+function useConsultationQuery(consultationId: any, id: any) {
+  return useQuery(routes.getDailyReport, {
+    pathParams: {
+      consultationId,
+      id,
+    },
+  });
+}
+
 export const DailyRoundListDetails = (props: any) => {
   const { facilityId, patientId, consultationId, id } = props;
-  const dispatch: any = useDispatch();
   const [dailyRoundListDetailsData, setDailyRoundListDetails] =
     useState<DailyRoundsModel>({});
   const [isLoading, setIsLoading] = useState(false);
```

Comment:
These states can be removed when using useQuery. You can use the `loading` from the `useQuery` return value instead of `isLoading` useState and `dailyRoundListDetailsData` is the same as `data` returned by `useQuery`.

`useQuery` handles most of the state management for you!